[2025-10-23T20:57:14.094+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-23T20:57:14.116+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v2.greet manual__2025-10-23T20:55:09.903738+00:00 [queued]>
[2025-10-23T20:57:14.129+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v2.greet manual__2025-10-23T20:55:09.903738+00:00 [queued]>
[2025-10-23T20:57:14.130+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 6
[2025-10-23T20:57:14.149+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): greet> on 2025-10-23 20:55:09.903738+00:00
[2025-10-23T20:57:14.157+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2405) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-23T20:57:14.160+0000] {standard_task_runner.py:72} INFO - Started process 2406 to run task
[2025-10-23T20:57:14.160+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v2', 'greet', 'manual__2025-10-23T20:55:09.903738+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpt7n10k8d']
[2025-10-23T20:57:14.162+0000] {standard_task_runner.py:105} INFO - Job 53: Subtask greet
[2025-10-23T20:57:14.253+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v2.greet manual__2025-10-23T20:55:09.903738+00:00 [running]> on host c65f76f8d89c
[2025-10-23T20:57:14.399+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineer' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v2' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2025-10-23T20:55:09.903738+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-23T20:55:09.903738+00:00'
[2025-10-23T20:57:14.401+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-23T20:57:14.404+0000] {logging_mixin.py:190} INFO - Hello World! My name is None None and I am 22 years old!
[2025-10-23T20:57:14.405+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-23T20:57:14.417+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-23T20:57:14.418+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v2, task_id=greet, run_id=manual__2025-10-23T20:55:09.903738+00:00, execution_date=20251023T205509, start_date=20251023T205714, end_date=20251023T205714
[2025-10-23T20:57:14.456+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-23T20:57:14.486+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-23T20:57:14.488+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
